packages/utils/date/getElapsedSince/getElapsedSince.ts

```diff
@@ -0,0 +1,68 @@
+type Props = {
+  targetDate: Date;
+  referenceDate?: Date;
+  language?: "kr" | "eng";
+  customLanguageObj?: {
+    justNow: string;
+    minutesAgo: string;
+    hoursAgo: string;
+    daysAgo: string;
+    monthsAgo: string;
+  };
+};
+
+/**
+ * This function returns a string representing the elapsed time since a given target date compared to a reference date.
+ *
+ * @param {Object} Props - The properties object.
+ * @param {Date} Props.targetDate - The target date to calculate the elapsed time from.
+ * @param {Date} [Props.referenceDate=new Date()] - The reference date to calculate the elapsed time against. (default: new Date())
+ * @param {string} [Props.language="kr"] - The language to use for the time strings. Choose between "kr" and "eng". (defaults: "kr").
+ * @param {Object} [Props.customLanguageObj] - Custom language object containing time strings. If provided, overrides the default language strings.
+ */
+export default function getElapsedSince({
+  targetDate,
+  referenceDate = new Date(),
+  language = "kr",
+  customLanguageObj,
+}: Props) {
+  const languageObj = {
+    kr: {
+      justNow: "방금",
+      minutesAgo: "분 전",
+      hoursAgo: "시간 전",
+      daysAgo: "일 전",
+      monthsAgo: "달 전",
+    },
+    eng: {
+      justNow: "now",
+      minutesAgo: "minutes ago",
+      hoursAgo: "hours ago",
+      daysAgo: "days ago",
+      monthsAgo: "months ago",
```

Comment:
It'd be nice if we can provide accurate plurality as well based on the time value.
Ex: `1 minute ago`, `2 minutes ago`

Reply:
I think additional work will be needed. Korean isn't a problem, but need to consider how plurals are expressed in other countries and with different custom labels.